sacrebleu/utils.py

```diff
@@ -27,6 +26,194 @@
 
 sacrelogger = logging.getLogger('sacrebleu')
 
+
+class Color:
+    ENABLE_COLORS = True
+    @staticmethod
+    def format(msg: str, color: str) -> str:
+        """Returns a colored version of the given message string.
+
+        :param msg: The string to Color.format.
+        :param color: The color specifier i.e. 'red', 'blue', 'green', etc.
+        :return: A colored version of the string if the output is a terminal.
+        """
+        if not Color.ENABLE_COLORS:
+            return msg
+        _ansi_str = getattr(colorama.Fore, color.upper(), None)
+        if _ansi_str:
+            return f'{_ansi_str}{msg}{colorama.Style.RESET_ALL}'
+
+        return msg
+
+
+def _format_score_lines(scores: dict,
+                        width: int = 2,
+                        multiline: bool = True) -> Dict[str, List[str]]:
+    """Formats the scores prior to tabulating them."""
+    new_scores = {'System': scores.pop('System')}
+    p_val_break_char = '\n' if multiline else ' '
+    is_bootstrap = False
+
+    def _color_p_value(p: float):
+        msg = f'(p = {p:.4f})'
+        if p > 0.05:
+            return Color.format(msg, 'red')
+        return msg + '*'
+
+    for metric, vals in scores.items():
+        new_vals = []
+
+        for result in vals:
+            if not isinstance(result, str):
+                # Format result instances
+                _str = f'{result.score:.{width}f}'
+                if result.mean is not None:
+                    is_bootstrap = True
+                    _str += f' / {result.mean:.{width}f} / {result.ci:.2f}'
+                if result.p_value is not None:
+                    _str += p_val_break_char + _color_p_value(result.p_value)
+            else:
+                # Already formatted in non paired-test mode
+                _str = result
+
+            new_vals.append(_str)
+
+        if is_bootstrap:
+            # Change titles
+            metric += ' / μ / ± 95% CI'
+
+        new_scores[metric] = new_vals
+
+    return new_scores
+
+
+def print_results_table(results: dict, signatures: dict, args: Namespace):
+    """Prints out a nicely formatted table for multi-system evaluation mode."""
+
+    tablefmt = args.format
+    if tablefmt in ('text', 'json'):
+        # Fallback to simple table if json is given
+        tablefmt = 'grid'
```

Comment:
What about `fancy_grid`?
We are already using unicode (μ / ±).

Reply:
It was `fancy_grid` actually but after adding `json`, I probably falled back to `grid` for some reason. Thanks.

Reply:
fixed